src/addressBook/index.js

```diff
@@ -342,3 +349,14 @@ export async function saveContact(addressBookIri, schema, fetch) {
   if (savePeopleError) return error(savePeopleError);
   return respond({ iri, contact, people });
 }
+
+export async function deleteContact(contactToDelete, fetch) {
+  const contactIri = contactToDelete.iri;
+  const contactContainerIri = contactIri.substring(
+    0,
+    contactIri.lastIndexOf("/") + 1
+  );
+
+  await deleteFile(contactIri, { fetch });
+  await deleteFile(contactContainerIri, { fetch });
```

Comment:
Could these be run in parallel with Promise.all?

Reply:
The link to the contact also needs to be removed from `people.ttl`.

Reply:
I think the latter will fail if for some reason it is resolved by the server before the former. So I think they need to be serialized.

The request to delete it from the People Index can be done parallell with any of them though.

Reply:
deleting a non-empty folder throws an error, so unfortunately no it seems.

and yep, until people.ttl was fixed we couldn't actually identify the entry to remove — can do this now though

Reply:
> The link to the contact also needs to be removed from `people.ttl`.

I think that bit will be easier to do after/along with https://inrupt.atlassian.net/browse/SOLIDOS-503. This does now update people.ttl though!